src/git.ts

```diff
@@ -87,6 +87,13 @@ export class Git {
 
   async pushChanges({ branch, message }: { branch: string; message: string }) {
     if (this.octokit) {
+      /** 
+       * Only add files form the current working directory
+       * 
+       * This will emulate the behavior of `git add .`,
+       * used in {@link commitAll}.
+       */
+      const addFromDirectory = process.cwd();
```

Comment:
We should add `cwd` as an option in the constructor and perhaps even pass it explicitly to all `exec` calls.

Reply:
Would this be instead of or in addition to calling `process.chdir()` here: 

https://github.com/changesets/action/blob/main/src/index.ts#L18-L22

Given that the currently released version has this discrepancy with usage of `git add`, I'd be inclined to do that refactor in a follow-up pull request? Just to limit risk of introducing more bugs with this bugfix PR :) What do you think?

Reply:
Ah okay cool, yeah I was thinking if we were to also remove the call to `chdir()` in the same PR. I'll make those changes shortly then! 👍

Reply:
Opened #475 to track removal of `process.chdir()`, there will need to be some work in `ghcommit` to account for some edge cases.